dev/integration_tests/android_embedding/pubspec.yaml

```diff
@@ -0,0 +1,72 @@
+name: android_embedding
+description: A new Flutter project.
+
```

Comment:
add `publish_to: none`

Reply:
I'd also generally just remove the comments/unnecessary bits from this file if possible.

Reply:
Done